describe.js

```diff
@@ -1,4 +1,4 @@
-(function(scope) {
+(function describeFactory(scope, returnInstance) {
```

Comment:
No, the factory could be called the text context, right?

I just don't like things named factory.